api/src/integration/java/org/pmiops/workbench/AbsorbAcceptanceTest.java

```diff
@@ -2,58 +2,73 @@
 import org.pmiops.workbench.google.DirectoryServiceImpl;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Import;
 
 public class AbsorbAcceptanceTest extends BaseIntegrationTest {
   @Autowired private CloudStorageClient cloudStorageClient;
 
+  @Autowired private AbsorbService absorbService;
+
+  private final String partiallyCompleteUserEmail =
+      "absorb_acceptance_test_02@fake-research-aou.org";
+  private final String nonexistantUserEmail = "absorb_acceptance_test_fake@fake-research-aou.org";
+
   @TestConfiguration
   @ComponentScan(basePackageClasses = DirectoryServiceImpl.class)
+  @Import({AbsorbServiceImpl.class})
   static class Configuration {}
 
   @Test
   @Disabled("RW-11039")
-  public void testGetCertificates() throws Exception {
-    // Setup: A user with this email exists and has completed both Absorb trainings in the
-    // test environment.
-    var email = "absorb_acceptance_test@fake-research-aou.org";
-
-    var config = cloudStorageClient.getAbsorbCredentials();
-    var apiKey = config.getString("apiKey");
-
-    // Obtain an access token
-    var accessToken =
-        new AuthenticateApi()
-            .restAuthenticationAuthenticate(
-                new AuthenticationRequest()
-                    .password(config.getString("password"))
-                    .username(config.getString("username"))
-                    .privateKey(apiKey),
-                apiKey);
-
-    // Find the user id
-    var users = new UsersApi().usersGetUsers(apiKey, accessToken, "username eq '" + email + "'");
-    assertThat(users.getUsers().size()).isEqualTo(1);
-    assertThat(users.getUsers().get(0).getUsername()).isEqualTo(email);
-
-    // Validate the user has completed two courses
-    var enrollments =
-        new EnrollmentsApi()
-            .enrollmentsGetUserEnrollments(
-                apiKey, accessToken, users.getUsers().get(0).getId(), null);
-    assertThat(enrollments.getEnrollments().size()).isEqualTo(2);
-    for (var enrollment : enrollments.getEnrollments()) {
-      assertThat(enrollment.getDateCompleted()).isNotNull();
-    }
+  public void testUserHasLoggedIntoAbsorb_True() throws Exception {
+    // Setup:
+    // - A user with this email exists
+    // - The user has logged into Absorb
+    assertThat(absorbService.userHasLoggedIntoAbsorb(partiallyCompleteUserEmail)).isTrue();
+  }
+
+  @Test
+  @Disabled("RW-11039")
+  public void testUserHasLoggedIntoAbsorb_False() throws Exception {
+    // Setup:
+    // - A user with this email does not exist, therefore they have not logged into Absorb
+    assertThat(absorbService.userHasLoggedIntoAbsorb(nonexistantUserEmail)).isFalse();
+  }
+
+  @Test
+  @Disabled("RW-11039")
+  public void testGetEnrollments() throws Exception {
```

Comment:
Do we need a test for a happy path? Both RT and CT done

Reply:
I don't think so, but it depends on what we want to use these "acceptance tests" for. I was trying to document Absorb's API behavior for courses that are complete and incomplete. Using RT and CT is a test implementation detail, I would expect any Absorb course to behave the same.